### PR TITLE
Update tableau.md

### PR DESCRIPTION
**Pull request by rpourzand**
Adding mention of entities, saved queries, and calculated field updates

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 1]
```diff
@@ -59,12 +59,14 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
...
    - To display available metrics and dimensions, dbt Semantic Layer returns metadata for a fake table with the dimensions and metrics as 'columns' on this table. Because of this, you can't actually query this table for previews or extracts. 
-   - Since this is treated as a table, dbt Semantic Layer can't dynamically change what is available. This means we display _all_ available metrics and dimensions even if a particular metric and dimension combination isn't available. 
-   
 - Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)
 - In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
+- We support calculated fields to the extent that you are using them to create a parameter filter or dynamically select metrics and dimensions, but other calculated field functions are not supported.
```
Comment (rpourzand) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]: This sounds super awkward. @aiguofer trying to figure out if I've captured the gist of it.

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 2]
```diff
@@ -45,9 +45,9 @@ This integration provides a live connection to the dbt Semantic Layer through Ta
 
 ## Using the integration
 
-1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page with all the metrics and dimensions configured in your dbt Semantic Layer. 
+1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to chooset between all the metrics, dimensions, entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](https://docs.getdbt.com/docs/build/saved-queries) that you've defined. These will also show up as data sources when you log in with the connector.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](https://docs.getdbt.com/docs/build/saved-queries) that you've defined. These will also show up as data sources when you log in with the connector.
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 3]
```diff
@@ -45,9 +45,9 @@ This integration provides a live connection to the dbt Semantic Layer through Ta
 
 ## Using the integration
 
-1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page with all the metrics and dimensions configured in your dbt Semantic Layer. 
+1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](https://docs.getdbt.com/docs/build/saved-queries) that you've defined. These will also show up as data sources when you log in with the connector.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](/docs/build/saved-queries) that you've defined. These will also show up as data sources when you log in with the connector.
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 4]
```diff
@@ -45,9 +45,9 @@ This integration provides a live connection to the dbt Semantic Layer through Ta
 
 ## Using the integration
 
-1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page with all the metrics and dimensions configured in your dbt Semantic Layer. 
+1. **Authentication** &mdash; Once you authenticate, the system will direct you to the data source page that allows you to choose between all the metrics, dimensions, and entities configured in your dbt Semantic Layer under the data source called "ALL". You can optionally access individual [saved queries](/docs/build/saved-queries) that you've defined. These will also show up as data sources when you log in with the connector.
 2. **Access worksheet** &mdash; From there, go directly to a worksheet in the bottom left-hand corner.
-3. **Access metrics and dimensions** &mdash; Then, you'll find all the metrics and dimensions that are available to query on the left side of your window.
+3. **Access metrics and dimensions** &mdash; Then, you'll find all the metrics, dimensions, entities that are available to query on the left side of your window.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
3. **Access metrics and dimensions** &mdash; Then, you'll find all the metrics, dimensions, and entities that are available to query on the left side of your window.
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 5]
```diff
@@ -59,12 +59,14 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
 ## Things to note
 
 - All metrics use the "SUM" aggregation type, and this can't be altered. The dbt Semantic Layer controls the aggregation type and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" (even though "SUM" is Tableau's default).
-- Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). 
+-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). For smaller slices of data that can be joined together, you can access saved queries.
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). You can use saved queries for smaller pieces of data that you want to combine.
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/use-dbt-semantic-layer/tableau.md]: what about:

> We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported.

**Comment by mirnawong1** [comment]
hey @rpourzand - reviewed it but left a sugg and waiting to hear what @aiguofer says! 🙏

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 6]
```diff
@@ -59,12 +59,14 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
 ## Things to note
 
 - All metrics use the "SUM" aggregation type, and this can't be altered. The dbt Semantic Layer controls the aggregation type and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" (even though "SUM" is Tableau's default).
-- Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). 
+-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). You can use saved queries for smaller pieces of data that you want to combine.
```
Comment (aiguofer) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md:65]: Might be worth mentioning that filters from saved queries will be applied to all queries using it.

File: website/docs/docs/use-dbt-semantic-layer/tableau.md (hunk 1 above)
Comment (aiguofer) [reply website/docs/docs/use-dbt-semantic-layer/tableau.md]: I like Mirna's suggestion but I would change `calculated field functions` to `uses of calculated fields`.

I wonder if we want to mention something like "if you have use cases for calculated fields that are not currently covered, please reach out so we understand the use case to see if we can support it".

There's probably a lot of use cases that we're not even aware of that might be totally legitimate and doable.

Comment (rpourzand) [reply website/docs/docs/use-dbt-semantic-layer/tableau.md]: That's a good idea! @mirnawong1 the "Things to note section" is getting kind of large. I'm going to try and break it up a bit and add these changes

**Comment by rpourzand** [comment]
@mirnawong1 @aiguofer I updated some more information here based on suggestions and broke out the "Things to note" Section - let me know your thoughts

@aiguofer  I also added the note about ALL vs METRICS AND DIMENSIONS

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 7]
```diff
@@ -56,15 +58,21 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
 - **From Desktop to Server** &mdash; Like any Tableau workflow, you can publish your workbook from Tableau Desktop to Tableau Server. For step-by-step instructions, visit Tableau's [publishing guide](https://help.tableau.com/current/pro/desktop/en-us/publish_workbooks_share.htm).
 
 
-## Things to note
+## Things to Note
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]: sentence case to match our style guide 🙏 
```suggestion
## Things to note
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 8]
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
...
-- Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). 
-   - To display available metrics and dimensions, dbt Semantic Layer returns metadata for a fake table with the dimensions and metrics as 'columns' on this table. Because of this, you can't actually query this table for previews or extracts. 
-   - Since this is treated as a table, dbt Semantic Layer can't dynamically change what is available. This means we display _all_ available metrics and dimensions even if a particular metric and dimension combination isn't available. 
-   
+**Aggregation**
+All metrics are shown as using the "SUM" aggregation type in Tableau's UI, and this cannot be altered using Tableau's interface. The dbt Semantic Layer controls the aggregation type in code and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" ("SUM" is Tableau's default).
+
+### Data Sources and Display
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
### Data sources and display
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 9]
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
...
-   
+**Aggregation**
+All metrics are shown as using the "SUM" aggregation type in Tableau's UI, and this cannot be altered using Tableau's interface. The dbt Semantic Layer controls the aggregation type in code and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" ("SUM" is Tableau's default).
+
+### Data sources and display
+-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). You can use saved queries for smaller pieces of data that you want to combine.
+- To display available metrics and dimensions, dbt Semantic Layer returns metadata for a fake table with the dimensions and metrics as 'columns' on this table. Because of this, you can't actually query this table for previews or extracts.
+
+### Calculations & Querying
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
### Calculations and querying
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 10]
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
...
+- To display available metrics and dimensions, dbt Semantic Layer returns metadata for a fake table with the dimensions and metrics as 'columns' on this table. Because of this, you can't actually query this table for previews or extracts.
+
+### Calculations and querying
 - Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)
 - In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
+- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported. _Note: If you have use cases for calculated fields that are not currently covered, please reach out to dbt Support we understand the use case_
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported. _Note: For calculated fields use cases that are not currently covered, please reach out to [dbt Support](mailto:support@getdbt.com) and share them so we can further understand._
```

File: website/docs/docs/use-dbt-semantic-layer/tableau.md [hunk 11]
```diff
@@ -58,13 +60,19 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
...
+- To display available metrics and dimensions, dbt Semantic Layer returns metadata for a fake table with the dimensions and metrics as 'columns' on this table. Because of this, you can't actually query this table for previews or extracts.
+
+### Calculations and querying
 - Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)
 - In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
+- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported. _Note: For calculated fields use cases that are not currently covered, please reach out to [dbt Support](mailto:support@getdbt.com) and share them so we can further understand._
```
Comment (mirnawong1) [inline website/docs/docs/use-dbt-semantic-layer/tableau.md]:
```suggestion
- We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other uses of calculated fields are not supported. _Note: For calculated fields use cases that are not currently covered, please reach out to [dbt Support](mailto:support@getdbt.com) and share them so we can further understand._
```

**Review by mirnawong1 — APPROVED** [review]
hey @rpourzand and @aiguofer - this looks good to me! are we ok to merge?

**Comment by mirnawong1** [comment]
hey team, fixed up the topics under 'things to note' - is this good to go?

Screenshot 2024-01-30 at 11 10 14

**Pull request merged**